packages/wdio-logger/src/index.ts

```diff
@@ -215,6 +226,36 @@ getLogger.setLogLevelsConfig = (logLevels: Record<string, log.LogLevelDesc> = {}
         loggers[logName].setLevel(logLevel)
     })
 }
+getLogger.setMaskingPatterns = (maskingPatternsStringPerLoggerName: Record<string, string | undefined> = {}, wdioDefault = undefined) => {
+
+    /**
+     * set the environment variable for masking patterns
+     */
+    if (process.env.WDIO_LOG_MASKING_PATTERNS === undefined) {
```

Comment:
This should be good enough and avoids cases where users may provide an empty string by accident
```suggestion
    if (!process.env.WDIO_LOG_MASKING_PATTERNS) {
```